gateway/mw_go_plugin.go

```diff
@@ -61,6 +61,12 @@ func (w *customResponseWriter) WriteHeader(statusCode int) {
 	w.ResponseWriter.WriteHeader(statusCode)
 }
 
+func (w *customResponseWriter) Flush() {
+	if flusher, ok := w.ResponseWriter.(http.Flusher); ok {
+		flusher.Flush()
+	}
+}
+
```

Comment:
Unrelated but safe. It particularly flushes some grpc buffers, but this isn't a grpc use case.